resources/frontend-development.md

```diff
@@ -8,3 +8,9 @@ Here is a list of useful websites for learning and mastering CSS:
 - [freeCodeCamp](https://www.freecodecamp.org/learn/)
 - [Flexbox Frogy](https://flexboxfroggy.com/)
 - [CSS Grid Garden](https://cssgridgarden.com/)
+
+## JavaScript
```

Comment:
@RitaDee, because JavaScript has some frameworks and libraries, let's add an `h3` here as per [this suggestion](https://github.com/shesharpnl/knowledge-hub/pull/51#discussion_r1450079937) to give clarification. 👍

```suggestion
## JavaScript

### React
```

Reply:
Do you mean I should create a sub-title ```JavaScript Library``` in a h3 tag?

Reply:
> Just put "React" as an h3. 👍

Okay. Thank you.